src/Adapter/Product/Update/ProductDuplicator.php

```diff
@@ -1087,12 +1087,12 @@ private function getRows(string $table, array $criteria, int $errorCode, array $
                 $arrayType = is_int(reset($value)) ? Connection::PARAM_INT_ARRAY : Connection::PARAM_STR_ARRAY;
                 $qb
                     ->andWhere("$column IN (:$column)")
-                    ->setParameter(":$column", $value, $arrayType)
+                    ->setParameter($column, $value, $arrayType)
                 ;
             } else {
                 $qb
                     ->andWhere("$column = :$column")
-                    ->setParameter(":$column", $value)
+                    ->setParameter($column, $value)
```

Comment:
I don't understand how it worked before. I had an error `Named parameter "id_product" does not have a bound value. `after update, in https://github.com/doctrine/dbal/blob/195aad31e9596e4093d417583f98342b6f0ec986/src/ArrayParameters/Exception/MissingNamedParameter.php#L11

Reply:
Yep the comma should only be added in the where, that was a mistake Maybe before Doctrine automatically understood it for some weird reason 😅